backend/routes/api/resources.js

```diff
@@ -4,9 +4,42 @@ const Resource = require("../../models/Resource");
 const errorWrap = require("../../utils/error-wrap");
 const { celebrate, Joi } = require("celebrate");
 const Fuse = require("fuse.js");
+const fetch = require("node-fetch");
 const { sortByDistance, options } = require("../../utils/resource-utils");
 Joi.objectId = require("joi-objectid")(Joi);
 
+async function addressToLatLong(address) {
+  let api_latlong = `http://www.mapquestapi.com/geocoding/v1/address?key=AAAppgYd4dZiZKW2sAtBtgEUAYejofok&maxResults=5&outFormat=json&location=${address}`;
```

Comment:
can we move the key to `.env`

Reply:
and maybe also the `api_latlong` to some constants section

Reply:
Agreed!